### PR TITLE
HKD-40 Time Sync Bar

### PR DESCRIPTION
**Pull request by koskelav**
React component for time synchronisation footer bar and saving manual input

Note: not yet accessing current video time for synchronisation

### CODE REVIEW

File: WebAPI/WebComponents/Meeting/src/SyncBar.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  const getCurrentVideoPosition = () => {
+    const videoPosition = 0;
+    setInputValue(convertToMMSS(videoPosition))
+  }
+
+  const convertToSec = () => {
+    var array = inputValue.split(":")
+    return parseInt(array[0] * 60) + parseInt(array[1])
+  }
+  const convertToMMSS = (seconds) => {
```
Comment (vijupa) [inline WebAPI/WebComponents/Meeting/src/SyncBar.js:32]: empty line between functions

File: WebAPI/WebComponents/Meeting/src/SyncBar.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const { meetingId } = props
+
+  const submitSync = (videoposition) => {
+    const request = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer ' + localStorage.getItem("userToken")
+      },
+      body: JSON.stringify({
```
Comment (vijupa) [inline WebAPI/WebComponents/Meeting/src/SyncBar.js:14]: Timestamp is not set here?

Comment (koskelav) [reply WebAPI/WebComponents/Meeting/src/SyncBar.js:15]: timestamp added as ISOString

Comment (vijupa) [reply WebAPI/WebComponents/Meeting/src/SyncBar.js:15]: It should be start time of the agenda point 2. We do not get the timestamp from the backend yet. So I think we should postpone merging this PR until we have the correct timestmap

**Pull request merged**